src/server/pfs/s3/s3.go

```diff
@@ -64,6 +64,10 @@ func (c *controller) requestClient(r *http.Request) *client.APIClient {
 	vars := mux.Vars(r)
 	if vars["s3gAuth"] != "disabled" {
 		accessKey := vars["authAccessKey"]
+		if strings.HasPrefix(accessKey, "PAC1") {
+			vars["isProjectAware"] = "yes"
```

Comment:
ok one more nit: should the expected value be "true" instead of "yes"?

Reply:
It’s a raw string, so I don’t think that the value matters so much. _But_ you make a good point: this ends up being a magic string and someone might set it to "true" or "YES" or "AFFIRMATIVE" or "yeah" or whatever. I replaced the magic strings with some constants.